modules/ducktests/tests/ignitetest/services/ignite.py

```diff
@@ -46,13 +46,40 @@ def __init__(self, context, config, num_nodes, jvm_opts=None, modules=None):
 
     # pylint: disable=W0221
     def start(self, timeout_sec=180):
+        self._rotate_log()
+
         super().start()
 
         self.logger.info("Waiting for Ignite(s) to start...")
 
         for node in self.nodes:
             self.await_node_started(node, timeout_sec)
 
+    def run(self, timeout_sec=180):
```

Comment:
Don't like it. Ducktape Service provides `start` method and it do some additional work than `start_node`. It is misleading which method (run or start) to invoke. As you add rotate log to start, can you remove run method at all?

Reply:
Unfortunatelly, ducktape allows start node only with cleaning. We should add smth like restart and comment that this should be rewrittten after appropriate change in ducktape codebase will be merged.

Reply:
Restart method with doc is ok. Let's do it this way.

Reply:
done